ush/python/pygfs/jedi/jedi.py

```diff
@@ -117,14 +201,15 @@ def execute(self) -> None:
         None
         """
 
+        # TODO: not sure if this chdir does anyththing
         chdir(self.jedi_config.rundir)
```

Comment:
I definitely noticed a case where for whatever reason (don't remember which machines/case(s), the runtime directory was not correct for the application being called unless this was done.